README.md

```diff
@@ -22,6 +22,70 @@ export default defineConfig({
 })
 ```
 
+## Options
+
+### `imports`
+
+Configures validation of import specifiers that should only be present in certain environments. Validation is performed against the raw import specifier in the source code.
+
+```ts
+{
+  imports?: {
+    server?: Array<string | RegExp>
+    client?: Array<string | RegExp>
+  }
+}
+```
+
+For example:
+
+```ts
+// vite.config.ts
+import { defineConfig } from "vite"
+import envOnly from "vite-env-only"
+
+export default defineConfig({
+  plugins: [
+    envOnly({
+      imports: {
+        server: ["fs-extra"],
+      },
```

Comment:
What about flipping it so that we specify what things _aren't_ allowed in a specific environment:

```suggestion
      denyImports: {
        client: ["fs-extra"],
      },
```

Otherwise, if there was something we wanted to exclude from the client we'd have to duplicate it:

```ts
      denyImports: {
        client: [],
        server: ["fs-extra"],
        workerd: ["fs-extra"],
        blah: ["fs-extra"],
        rsc: ["fs-extra"],
        // ....
      },
```

Reply:
I love this. Makes it so much clearer.